Source/WebCore/page/Navigation.cpp

```diff
@@ -548,23 +548,29 @@ void Navigation::cleanupAPIMethodTracker(NavigationAPIMethodTracker* apiMethodTr
 // https://html.spec.whatwg.org/multipage/nav-history-apis.html#abort-the-ongoing-navigation
 void Navigation::abortOngoingNavigation(NavigateEvent& event)
 {
+    auto* globalObject = scriptExecutionContext()->globalObject();
+    if (!globalObject && m_ongoingAPIMethodTracker)
+        globalObject = m_ongoingAPIMethodTracker->committedPromise->globalObject();
+    ASSERT(globalObject);
```

Comment:
```suggestion
```

Reply:
I think the ASSERt is reasonable, since it really should not happen AFAICS.